man/man5/zfs-module-parameters.5

```diff
@@ -1637,6 +1637,26 @@ OpenZFS pre-release versions and now have compatibility issues.
 Default value: \fB0\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_forced_export_unmount\fR (int)
+.ad
+.RS 12n
+During forced unmount, leave the filesystem in a disabled mode of operation,
+in which all new I/Os fail, except for those required to unmount it.
```

Comment:
Is the alternative (the default) that a forced unmount will fail if there are i/o's in progress?  by "i/o's in progress" do we mean that there's a ZPL syscall (e.g. read/write) in progress with a zio_t outstanding?

Why would we not want the default to be `zfs_forced_export_unmount=1`?  What's the downside?

Reply:
There's no downside, IMO, but Linux has a long-standing practice of disallowing this on other filesystems, so I created the tunable to conform to that.